### DISCUSSION TITLE
Several questions about the size, deletion, and modification of the vectors

### DISCUSSION THREAD

**Discussion by coollofty** [milvus-io/milvus#4869] [Q&A and General discussion]
1、集合会被载入显存，如果我在机器人上使用了N多个集合，也就是说显存会爆掉？程序运行时会出错？
2、一个集合设置为512MB索引，也就是512MB一个文件，某个已经建立了索引，再删除刚好位于这个文件中的数据，索引还会重建？是否新加数据永远都是在新的分片文件中？
3、似乎没有更新向量数据的接口？那就只能是删除了重新添加？

**Comment by JackLCL** [comment]
@coollofty 
1、集合里的数据都是进行了分片的，程序运行不会出错；
2、不会重建。是的。
3、没有update的接口，更新目前只能先删除再插入。

如果还有其他问题，欢迎加入我们的技术交流群一起讨论~
![image](https://user-images.githubusercontent.com/53512883/112827292-b897aa80-90c0-11eb-80fd-ab9b024a1977.png)
